### ISSUE TITLE
Calling encode_from_hooked causes error after recent commit

### ISSUE THREAD

**Issue by NilsLattek** [ueberauth/guardian#247]
Commit 1deca7f8cb11ca8918892e977444599d7ac4f37b changed the pattern matching in guardian.ex line 100 from ```:ok``` to ```{:ok, _}```:
```elixir
case call_after_encode_and_sign_hook(
      resource,
      type,
      claims_from_hook, jwt
) do
      {:ok, _} -> {:ok, jwt, claims_from_hook}
      {:error, reason} -> {:error, reason}
end
```

But the default return value of the ```after_encode_and_sign``` is still just ```:ok```, so we get the error ```no case clause matching: :ok```.
hooks.ex:
```def after_encode_and_sign(_, _, _, _), do: :ok```

**Comment by geofflane** [comment]
+1 Seeing this as well.

**Comment by blatyo** [comment]
Could you also retire this version or release a new version? Right now, if you don't fix the version to 0.14.0, you'll hit the issue.

https://hex.pm/docs/tasks#hex_retire

**Comment by doomspork** [comment]
0.14.2 has been released.  I'll retire 0.14.1 later today.
